### PR TITLE
Tinybike/ch8621/close position button on trade page not working

### PR DESCRIPTION
**Pull request by bthaile**
[Clubhouse Story](https://app.clubhouse.io/augur/story/8621)

Refactored market position list to handle multiple messages based on close position, created ch ticket to refactor to allow for informing user of partial position fills. 

## Submitter checklist
- [x] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [x] Steps to reproduce
  - [x] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass

### CODE REVIEW

**Comment by adrake33** [comment]
This pull request **introduces 1 alert** when merging 6dba49245fee33538ad9d5d251856f9aaa38cecf into ca922e404b35f5d506127903f9325d6e280eaf12 - [view on lgtm.com](https://lgtm.com/projects/g/AugurProject/augur/rev/pr-2b885883428639b9c095db6492e5246b349c7950)

**new alerts:**

* 1 for Potentially inconsistent state update

---

*Comment posted by [lgtm.com](https://lgtm.com)*

**Comment by adrake33** [comment]
This pull request **introduces 1 alert** when merging 5b33be7739600c3fe555f11b0334bcd77ae8e591 into ca922e404b35f5d506127903f9325d6e280eaf12 - [view on lgtm.com](https://lgtm.com/projects/g/AugurProject/augur/rev/pr-349b3f2fb5b0e94015c9edcb7d1b9e2559321ebf)

**new alerts:**

* 1 for Potentially inconsistent state update

---

*Comment posted by [lgtm.com](https://lgtm.com)*

**Comment by adrake33** [comment]
This pull request **introduces 1 alert** when merging 15b808e9878fd173e9a79d8ef05dd443217859ae into d3d758e68a416150615f9fb103316f6a82b5a1fa - [view on lgtm.com](https://lgtm.com/projects/g/AugurProject/augur/rev/pr-392cfd4597a64ced384de73aade0459eeb5ac0f9)

**new alerts:**

* 1 for Potentially inconsistent state update

---

*Comment posted by [lgtm.com](https://lgtm.com)*

**Comment by justinbarry** [comment]
screen shot 2018-05-11 at 12 45 12 pm

**Comment by justinbarry** [comment]
Waiting on travis ci build.

File: src/modules/market/containers/market-portfolio-card.js [hunk 1]
```diff
@@ -6,11 +6,16 @@ import { determineMarketLinkType } from 'modules/market/helpers/determine-market
 import MarketPortfolioCard from 'modules/market/components/market-portfolio-card/market-portfolio-card'
 import { selectMarket } from 'modules/market/selectors/market'
 import { sendFinalizeMarket } from 'modules/market/actions/finalize-market'
+import { calculateOutstandingReturns } from 'modules/market/helpers/calculate-outstanding-returns'
```
Comment (justinbarry) [inline src/modules/market/containers/market-portfolio-card.js]:
```
ERROR in ./src/modules/market/containers/market-portfolio-card.js
Module not found: Error: Can't resolve 'modules/market/helpers/calculate-outstanding-returns' in '/Users/justin/WebstormProjects/augur/src/modules/market/containers'
 @ ./src/modules/market/containers/market-portfolio-card.js 9:0-99
 @ ./src/modules/portfolio/components/positions-markets-list/positions-markets-list.jsx
 @ ./src/modules/portfolio/components/positions/positions.jsx
 @ ./src/modules/portfolio/containers/positions.js
 @ ./src/modules/portfolio/components/portfolio-view/portfolio-view.jsx
 @ ./src/modules/routes/constants/components.js
 @ ./src/modules/routes/components/routes/routes.jsx
 @ ./src/modules/app/components/app/app.jsx
 @ ./src/modules/app/containers/app.js
 @ ./src/main.jsx
 @ multi ./src/main
```

Comment (bthaile) [reply src/modules/market/containers/market-portfolio-card.js]: Yea that reference needs to be removed.

**Comment by bthaile** [comment]
Removed reference to fix compile issue.

**Comment by adrake33** [comment]
This pull request **introduces 1 alert** when merging da6371ec35851feb56c8633cbbda183e1c13d4ca into d3d758e68a416150615f9fb103316f6a82b5a1fa - [view on lgtm.com](https://lgtm.com/projects/g/AugurProject/augur/rev/pr-81c56e08cffdd8ede7659d4aac9e8003a06c935f)

**new alerts:**

* 1 for Potentially inconsistent state update

---

*Comment posted by [lgtm.com](https://lgtm.com)*

**Pull request merged**